src/_pytest/pathlib.py

```diff
@@ -51,6 +51,8 @@
     1921,  # ERROR_CANT_RESOLVE_FILENAME - fix for broken symlink pointing to itself
 )
 
+TMPDIR_FILE_MODE = int(os.getenv("PYTEST_TMPDIR_FILE_MODE", "0o700"), 8)
```

Comment:
If you specify base 8, then you can drop the `0o` prefix.

Reply:
I can, but should I? 

I felt it was clearer this way. But happy to remove it if you think it's better.

Reply:
To me the `0o` means that the field doesn't specify a base, i.e. `123` is decimal, `0x123` would work, etc. I think the base 8 is good, but then we should drop the `0o`